backend/backend.py

```diff
@@ -13,10 +13,11 @@
 @click.option("-t", "--thresholds-file", default="DefaultThresholds")
 @click.option("-d", "--debug", is_flag=True)
 @click.option("-c", "--concurrent-connections", type=int)
+@click.option("-p", "--password-dynamically", default=None)
```

Comment:
Let's change the variable name from password-dynamically to password. To let people know it's dynamic, we can add a help text https://click.palletsprojects.com/en/8.1.x/documentation/